app-cdr/cdrtools/cdrtools-3.01~a30.recipe

```diff
@@ -0,0 +1,165 @@
+	"
+LICENSE="
+	GNU GPL v2
+	CDDL v1
+	"
+SOURCE_URI="http://downloads.sourceforge.net/project/cdrtools/alpha/cdrtools-3.01a30.tar.bz2"
+CHECKSUM_SHA256="5b9a2f98771c9d0097a1e7640727655ece2864eea95f38e5611af2b2f6e6d9cd"
+SOURCE_DIR="cdrtools-3.01"
+PATCHES="cdrtools-3.01~a30.patch"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+GLOBAL_WRITABLE_FILES="
+	settings/cdrecord keep-old
+	settings/rscsi keep-old
+	"
+
+PROVIDES="
+	cdrtools = $portVersion compat >= 3
+	cmd:devdump = $portVersion compat >= 3
+	cmd:rscsi = $portVersion compat >= 3
+	cmd:scgcheck = $portVersion compat >= 3
+	cmd:mkisofs = $portVersion compat >= 3
+	cmd:btcflash = $portVersion compat >= 3
+	cmd:scgskeleton = $portVersion compat >= 3
+	cmd:isovfy = $portVersion compat >= 3
+	cmd:readcd = $portVersion compat >= 3
+	cmd:isodebug = $portVersion compat >= 3
+	cmd:cdda2mp3 = $portVersion compat >= 3
+	cmd:cdda2ogg = $portVersion compat >= 3
+	cmd:cdda2wav = $portVersion compat >= 3
+	cmd:mkhybrid = $portVersion compat >= 3
+	cmd:cdrecord = $portVersion compat >= 3
+	cmd:isodump = $portVersion compat >= 3
+	cmd:isoinfo = $portVersion compat >= 3
+	"
+
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	"
+
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:find
+	cmd:gcc
+	cmd:grep
+	cmd:ld
+	cmd:make
+	cmd:sed
+	"
+
+patchInsdir()
+{
+	# Usage: patchInsdir <oldDir> <newDir> <file> ...
+	oldDir=$1
+	newDir=$2
+	shift 2
+
+	sed -i "s,^INSDIR=\s*$oldDir,INSDIR= $newDir," $@
+}
+
+PATCH()
+{
+	allMakefiles="$(find . -name Makefile\*) $(find . -name \*.mk)"
+
+	patchInsdir bin $relativeBinDir $allMakefiles
+	patchInsdir sbin $relativeBinDir $allMakefiles
+	patchInsdir share/doc $relativeDocDir $allMakefiles
+	patchInsdir include $relativeIncludeDir $allMakefiles
+	patchInsdir lib $relativeLibDir $allMakefiles
+	patchInsdir etc/default settings $allMakefiles
+
+	sed -i "s,/etc/default,$sysconfDir," \
+		btcflash/btcflash.1 \
+		cdda2wav/cdda2mp3 \
+		cdda2wav/cdda2ogg \
+		cdda2wav/cdda2*.1 \
+		cdrecord/COPYING \
+		cdrecord/README.* \
+		cdrecord/auinfo.c \
+		cdrecord/cdrecord.1 \
+		doc/*.man \
+		doc/*.ps \
+		include/schily/deflts.h \
+		libcdrdeflt/cdrdeflt.c \
+		libcdrdeflt/cdrdeflt.h \
+		mkisofs/diag/isoinfo.8 \
+		readcd/readcd.1 \
+		rscsi/rscsi.1 \
+		rscsi/rscsi.c \
+		scgskeleton/scgskeleton.1
+}
+
+BUILD()
+{
+	# not multi-job safe
+	make GMAKE_NOWARN=true INS_BASE=$prefix INS_RBASE=$prefix LDPATH="" \
+		RUNPATH="" DEFMANBASE=$relativeDocumentationDir
+}
+
+INSTALL()
+{
+	make GMAKE_NOWARN=true INS_BASE=$prefix INS_RBASE=$prefix LDPATH="" \
+		RUNPATH="" DEFMANBASE=$relativeDocumentationDir install
+
+	# The whole lib folder only contains static libraries (even in several
+	# subdirectories).
+	mkdir -p $developLibDir
+	mv $libDir/* $developLibDir/
+	rmdir $libDir
+
+	# devel package
+	packageEntries devel \
+		$developDir
+}
+
+# ----- devel package -------------------------------------------------------
+
+SUMMARY_devel="cdrtools development files"
```

Comment:
I browsed several recipes (while looking for templates for cross-building) and noticed that many had the _devel sections at the end. Then I went through your comments to see if you had mentionned such moves but found none... I did recall having dropped a line starting with "# ----- devel package ------", so I came to the *wrong* conclusion that PROVIDES_devel and 
REQUIRES_devel were at the end...
I'll move them back to their first locations, in this order:
PROVIDES
REQUIRES
PROVIDES_devel
REQUIRES_devel
BUILD_REQUIRES
BUILD_PREREQUIRES

Is this order OK, or should I switch the order of REQUIRES and PROVIDES_devel to get this: 

PROVIDES
PROVIDES_devel
REQUIRES
REQUIRES_devel
BUILD_REQUIRES
BUILD_PREREQUIRES

Thanks @waddlesplash